### PR TITLE
Added ARM template to Deploy Drupal 8 VM Scaleset, with gluster FS and MySQL

### PR DESCRIPTION
**Pull request by maniSbindra**
### Changelog
* This pull request is to add new ARM template which Deploy Drupal 8 VM Scale set, with glusterfs and MySQL
* Gluster ARM template is used as a nested template to create the gluster cluster
* Template provides option to either to use an existing MySQL server for the Drupal database, or create a new server
* README has details of deployment and parameter configurations ### Description of the change

### Description of the change
* This Change adds new files and folders needed by the new Drupal 8 ARM template (with VM Scale set, glusterfs and MySQL

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@maniSbindra__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by azurecla** [comment]
__@maniSbindra__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

File: 301-drupal8-vmss-glusterfs-mysql/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,657 @@
...
+      "defaultValue": "Standard_D1_v2",
+      "metadata": {
+        "description": "Size of Drupal VMs in the VM Scale Set."
+      }
+    },
+    "drupalUbuntuOSVersion": {
+      "type": "string",
+      "defaultValue": "14.04.4-LTS",
+      "allowedValues": [
+        "15.10",
```
Comment (singhkay) [inline 301-drupal8-vmss-glusterfs-mysql/azuredeploy.json]: Remove 15.10, the image is removed from Azure.

File: 301-drupal8-vmss-glusterfs-mysql/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,657 @@
...
+    "drupalAdminPassword": {
+      "type": "securestring",
+      "defaultValue": "Drupal Administrator Password",
+      
+      "metadata": {
+        "description": "The Ubuntu version for the Drupal VM. This will pick a fully patched image of this given Ubuntu version. Allowed values are: 15.10, 14.04.4-LTS."
+      }
+    },
+    
+    "drupalVersion": {
```
Comment (singhkay) [inline 301-drupal8-vmss-glusterfs-mysql/azuredeploy.json]: This should be a variable if it only has one allowed value

**Comment by singhkay** [comment]
@maniSbindra Left some comments to address. Ideally the folder name should be 301-vmss-drupal8-glusterfs-mysql so that this template fits in with other VMSS templates

**Pull request merged**